src/gui/guiFormSpecMenu.cpp

```diff
@@ -3283,8 +3283,14 @@ void GUIFormSpecMenu::drawMenu()
 	bool hovered_element_found = false;
 
 	if (hovered != NULL) {
-		s32 id = hovered->getID();
+		if (m_show_debug) {
+			core::rect<s32> hovered_pos = hovered->getAbsolutePosition();
+			driver->draw2DRectangle(0x22FFFF00,
+				hovered->getAbsoluteClippingRect(),
+				&hovered_pos);
```

Comment:
Parameters are` color, pos, *clip`, not `color, clip, *pos`. http://irrlicht.sourceforge.net/docu/classirr_1_1video_1_1_i_video_driver.html#ac7f452fae0ef8abe01768a78ba7033b7
(Just drawing the clipping rect would also suffice.)

Reply:
Nice catch, thank you. I adjusted the PR accordingly.